RadarSDK/RadarUtils.m

```diff
@@ -32,7 +32,14 @@ + (NSNumber *)timeZoneOffset {
 }
 
 + (NSString *)sdkVersion {
-    return @"3.2.2";
+    NSBundle *bundle = [NSBundle bundleForClass:RadarUtils.class];
+    NSString *versionNumber = bundle.infoDictionary[@"CFBundleShortVersionString"];
```

Comment:
@jrtibbetts Very cool. Will this always pull the marketing version from the SDK's plist? Will that still work if the SDK is embedded in a customer's app? (Just want to make extra sure we'd never pull the customer's app version instead.)

Side note: It may actually be useful to pull the customer's app version and send it up to `POST /track` as `appVersion`. This could help with debugging. (When I'm debugging Radar, how do I know which SDK version _and_ app version the user is running?

cc @phil-brown @jsani-radar to consider the same thing for Android.

Reply:
If they're using the Radar SDK *as a framework*, it will always pull it from the Radar SDK framework. If they statically linked the Radar code into their app (i.e. without using a framework), it would be their app's version. Do we know whether any customers are using it that way?

Reply:
I don't think we can guarantee whether customers statically link or use a framework to integrate Radar - we'll have to support / be resilient to both. Would it be possible to namespace it such that it always grabs the Radar SDK version?

Reply:
FWIW this _is_ doable for Android.